### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by dat-lequoc** [brunocarmena57/ErasTourBot#1]
Hello folk,

I'm struggling to book Taylor Swift master ticket. Did this bot work? Can you explain a bit about your code please?

Thank you very much.

**Comment by brunocarmena57** [comment]
Hi!

The code only opens the tabs in the sale tickets store in Brazil using a asynchronous method. I tried making it to buy the tickets as well, but i couldn't because there was only one tickets sale here, and i struggled getting the html code before the opening... But if you manage to obtain this part before the sale in your country (like using a previous sale or taylor swift having more than one show there), you can use Selenium to automate all the tasks of buying the tickets! If you want help reformulating the code and adding that part, just let me know! ;)

**Comment by dat-lequoc** [comment]
Hi,

Thank you very much for your response. 
Indeed, I could have made the purchase this morning, but it was a pity that I had a Wi-Fi problem. 

Regardless, I searched for a Selenium script which could let me know when the queue opens. Now, I've figured out that a Chrome extension can get the job done instead. For buying a ticket, I believe I can do it manually. I'll give it my best shot tomorrow. :D

Thanks again for your help.
Dat
